.github/workflows/main.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -v --editable .[lint]
```

Comment:
```suggestion
          pip install -v --editable .[lint, test]
```

Reply:
To push coverage we also need to run tests, right?

Reply:
True, thanks! I forgot how coveralls.io works...

Reply:
The syntax seems a bit strange, if you want to install multiple extra dependencies.